src/queries/newsItems.js

```diff
@@ -1,7 +1,9 @@
 import gql from 'graphql-tag';
 
 export const GET_NEWS_ITEMS = gql`
-  query NewsItems($limit: Int, $offset: Int) {
+  query NewsItems($limit: Int, $offset: Int, $categoryId: ID) {
```

Comment:
in den queries muss vllt. `$categoryId: String` statt `$categoryId: ID`. wird sich zeigen, wenn server ready ist

Reply:
überall nun `$categoryId: ID`